src/components/Footer.jsx

```diff
@@ -14,7 +14,7 @@ const Footer = () => {
         <VersionLabel />
       </div>
       <div className="footer__center">
-        Copyright 2021{' '}
+        Copyright 2022{' '}
```

Comment:
{`Copyright ${dayjs().year()} `}

Będzie spokój za rok ;)
https://day.js.org/docs/en/get-set/year

Reply:
Pomysł ogólnie dobry. Zrobiłem to wbudowaną w react funkcją `new Date().getFullYear()`. Ta libka co poleciłeś ma tylko 2400 pobrań tygodniowo.